less/navigation.less

```diff
@@ -1 +1,46 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+@import "variables.less";
+
+header{
+
+    margin-top: 20px;
+
+        
+
+
+    
+    div{
+        display: flex;
+        justify-content: space-between;
+        max-width: 800px;
+        margin: 0 auto;
+
+        @media @mobile{
+            width: 400px;
+            flex-direction: column;
+            padding-right: 40px; 
+        }
+    
+    fun-header{
+        @media @mobile{
+            width: 400px;
```

Comment:
Adding a text-align to center the text would make it better for the layout and adding some padding to give it some more space also